runtime/src/main/java/io/quarkiverse/operatorsdk/runtime/StartupListener.java

```diff
@@ -24,7 +24,7 @@ public void onStartup(@Observes StartupEvent event) {
             final var branch = !version.getExtensionBranch().equals(Version.UNKNOWN)
                     ? " on branch: " + version.getExtensionBranch()
                     : "";
-            log.info("Quakus Java Operator SDK extension {} (commit: {}{})", version.getExtensionVersion(),
+            log.info("Quarkus Java Operator SDK extension {} (commit: {}{})", version.getExtensionVersion(),
```

Comment:
Quakus is the next great framework for duck programmers! 🤦🏼

Reply:
Oh, this is the story behind it... 😃